### PR TITLE
Allow tags for Accounts, Addresses and Contracts

### LINKED ISSUE #2643: Allow tagging of Accounts, Addresses, Contracts
- applies to accounts, address book, contracts
- store in metadata, `{ "tags": ["tag1", "tag2", "tag3" ...] }`
- tag via edit metadata (i.e. along with name, description edit functionality)
- search by tag
- sort by tag
- initial version is ok with just tagging and displaying it (search/sort as extensions)

From Slack:

```
gavofyork [2:11 PM]  
another miner feature request:
tags for accounts  
just your usual semantic tagging
then have them be search/sortable with the tags
```

### PR DESCRIPTION
**Pull request by ngotchac**
[Copy of PR #2697]

Closes #2643

This PR adds tagging capabilities to Accounts, Addresses and Contracts. Multiple can be added per item, and these can be searched. They can also be sorted by name or by tag.

Currently, the search accepts multiple tokens, that are joined with OR ; eg. if item 1 has tag foo and item 2 has tag bar, as search for the two tokens foo and bar will display item 1 and 2.
This can be changed easily to an AND behavior.

See it in action here: https://youtu.be/3UA-QI0G_UQ

### CODE REVIEW

File: js/webpack.config.js [hunk 1]
```diff
@@ -131,6 +132,16 @@ module.exports = {
         ]
       }),
       new HappyPack({
+        id: 'less',
```
Comment (jacogr) [inline js/webpack.config.js]: Do we still need this now?

Comment (ngotchac) [inline js/webpack.config.js]: So it's not used by any file, but it was there previously, so not sure...
Should we get completely rid of LESS ? (ie. in Webpack and `package.json`)

Comment (jacogr) [inline js/webpack.config.js]: If we are not using it, yes, best approach.

Comment (ngotchac) [inline js/webpack.config.js]: OK, will do

**Pull request merged**